NAMESPACE

```diff
@@ -104,6 +104,8 @@ if (getRversion() >= "4.0.0") {
   # version of R (and that is checked in .onLoad with error if not).
   export(.rbind.data.table) # only export in R<4.0.0 where it is still used; R-devel now detects it is missing doc, #5600
 }
+if (getRversion() >= "3.6.0") S3method(knitr::knit_print, data.table)
+# else manual delayed registration from the onLoad hook
```

Comment:
is this comment attached to the `knitr::knit_print` directive? If so either move to the end of the line or the previous line.